js/src/orange-navbar.js

```diff
@@ -0,0 +1,40 @@
+import EventHandler from './dom/event-handler'
+import BaseComponent from './base-component'
+import SelectorEngine from './dom/selector-engine'
+
+const NAME = 'orangenavbar'
+const DATA_KEY = 'bs.orangenavbar'
+const EVENT_KEY = `.${DATA_KEY}`
+const DATA_API_KEY = '.data-api'
+const EVENT_SCROLL_DATA_API = `scroll${EVENT_KEY}${DATA_API_KEY}`
+const SELECTOR_STICKY_TOP = 'header.sticky-top'
+
+class OrangeNavbar extends BaseComponent {
+  // Getters
+  static get NAME() {
+    return NAME
+  }
+
+  enableMinimizing(el) {
+    // The minimized behaviour works only if your header has .sticky-top (fixed-top will be sticky without minimizing)
+    const Scroll = window.scrollY
+    const headerChildren = [...el.children]
+    const globalHeaderChild = headerChildren.find(element => !element.classList.contains('supra'))
+    if (globalHeaderChild) {
+      if (Scroll > 0) {
+        // Consider first element not having .supra in array is the first header
+        globalHeaderChild.classList.add('header-minimized')
+      } else {
+        globalHeaderChild.classList.remove('header-minimized')
+      }
+    }
+  }
+}
+
+EventHandler.on(window, EVENT_SCROLL_DATA_API, () => {
+  for (const el of SelectorEngine.find(SELECTOR_STICKY_TOP)) {
+    OrangeNavbar.getOrCreateInstance(el).enableMinimizing(el)
+  }
+})
+
```

Comment:
Need to add jQueryInterface

Reply:
Here is the archive of the page I created to test that it works with jQuery.

```html
<!doctype html>
<html lang="en">
  <head>
    <!-- Required meta tags -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <!-- Preconnect to CDN: remove if not needed -->
    <link href="https://cdn.jsdelivr.net" rel="preconnect" crossorigin="anonymous">

    <!--
      Neue Helvetica is a trademark of Monotype Imaging Inc. registered in the U.S.
      Patent and Trademark Office and may be registered in certain other jurisdictions.
      Copyright © 2014 Monotype Imaging Inc. All rights reserved.
      Orange has purchased the right to use Helvetica in its websites and mobile applications.
      If you are not authorized to used it, don't include the orangeHelvetica.css
      See NOTICE.txt for more information.
    -->
    <!-- Option 1: Use a CDN -->
    <link href="https://cdn.jsdelivr.net/npm/boosted@5.1.3/dist/fonts/HelvNeue55_W1G.woff2" rel="preload" as="font" type="font/woff2" integrity="sha384-R6e0PFLMMV6HBvkQK22ecNfjOzyh89wSndiTC71MuvoaOnhIYgOAGVC0gW0kVN16" crossorigin="anonymous">
    <link href="https://cdn.jsdelivr.net/npm/boosted@5.1.3/dist/fonts/HelvNeue75_W1G.woff2" rel="preload" as="font" type="font/woff2" integrity="sha384-ylOkwNNvSwXpWNbpEhI45ruJTXyfQbIb42IxMvSGGcndZBpZ9iAmOFSUl4/Goeqz" crossorigin="anonymous">
    <link href="https://cdn.jsdelivr.net/npm/boosted@5.1.3/dist/css/orange-helvetica.min.css" rel="stylesheet" integrity="sha384-ARRzqgHDBP0PQzxQoJtvyNn7Q8QQYr0XT+RXUFEPkQqkTB6gi43ZiL035dKWdkZe" crossorigin="anonymous">

    <!-- Option 2: Embed the fonts
    <link href="dist/fonts/HelvNeue55_W1G.woff2" rel="preload" as="font" type="font/woff2" crossorigin="anonymous">
    <link href="dist/fonts/HelvNeue75_W1G.woff2" rel="preload" as="font" type="font/woff2" crossorigin="anonymous">
    <link href="dist/css/orange-helvetica.min.css" rel="stylesheet" crossorigin="anonymous">
    -->

    <!-- Boosted CSS -->
    <link href="../Orange-Boosted-Bootstrap/dist/css/boosted.min.css" rel="stylesheet">

    <title>Hello, world!</title>
  </head>
  <body style="height: 150vh;">
    <main style="height: 150vh;">
	  <h1 class="visually-hidden">Difference between sticky and fixed navbar</h1>
	  <!-- <main class="exclude-from-pa11y-analysis"> -->
	  <header class="sticky-top">
		<nav class="navbar navbar-dark navbar-expand bg-dark supra" aria-label="{{$aria_label}}">
		  <div class="container">
			<ul class="navbar-nav me-auto">
			  <li class="nav-item"><a href="#" class="nav-link active" aria-current="page">Personal</a></li>
			  <li class="nav-item"><a href="#" class="nav-link">Business</a></li>
			</ul>
			<ul class="navbar-nav">
			  <li class="nav-item"><a class="nav-link active" href="#" aria-label="English version">EN</a></li>
			  <li class="nav-item"><a class="nav-link" href="#" aria-label="Version française" lang="fr" hreflang="fr">FR</a></li>
			  <li class="nav-item"><a class="nav-link" href="#" aria-label="Version en español" lang="es" hreflang="es">ES</a></li>
			</ul>
		  </div>
		</nav>

		<nav class="navbar navbar-dark bg-dark navbar-expand-lg">
		  <div class="container">
			<a class="navbar-brand order-first me-auto me-lg-4" href="#">
				<img src="../Orange-Boosted-Bootstrap/site/content/docs/5.1/assets/brand/orange-logo.svg" width="50" height="50" role="img" alt="Boosted" loading="lazy">
			</a>
			<button class="navbar-toggler collapsed" type="button" data-bs-toggle="collapse" data-bs-target=".id" aria-controls="id.1 id.2" aria-expanded="false" aria-label="Toggle navigation">
			  <span class="navbar-toggler-icon"></span>
			</button>
			<div id="id.1" class="navbar-collapse collapse me-lg-auto id">
			  <ul class="navbar-nav border-top border-1 border-dark d-flex flex-row d-lg-none supra">
				<li class="nav-item"><a href="#" class="nav-link active" aria-current="page">Personal</a></li>
				<li class="nav-item"><a href="#" class="nav-link">Business</a></li>
			  </ul>
			  <ul class="navbar-nav">
				<li class="nav-item"><a class="nav-link" href="#">Label</a></li>
				<li class="nav-item"><a class="nav-link" href="#">Label</a></li>
				<li class="nav-item"><a class="nav-link" href="#">Label</a></li>
				<li class="nav-item"><a class="nav-link" href="#">Label</a></li>
				<li class="nav-item"><a class="nav-link" href="#">Label</a></li>
				<li class="nav-item"><a class="nav-link" href="#">Label</a></li>
			  </ul>
			  <ul class="navbar-nav border-top border-1 border-dark d-flex flex-row d-lg-none">
				<li class="nav-item"><a class="nav-link active" href="#" aria-label="English version">EN</a></li>
				<li class="nav-item"><a class="nav-link" href="#" aria-label="Version française" lang="fr" hreflang="fr">FR</a></li>
				<li class="nav-item"><a class="nav-link" href="#" aria-label="Version en español" lang="es" hreflang="es">ES</a></li>
			  </ul>
			</div>
			<div id="id.2" class="navbar-collapse collapse d-sm-flex id">
			  <ul class="navbar-nav flex-row">
				<li class="nav-item">
				  <a href="#" class="nav-link nav-icon">
					<svg width="1.5rem" height="1.5rem" fill="currentColor" aria-hidden="true" focusable="false" class="overflow-visible">
					  <use xlink:href="../Orange-Boosted-Bootstrap/site/static/docs/5.1/assets/img/boosted-sprite.svg#search" />
					</svg>
					<span class="visually-hidden">Search</span>
				  </a>
				</li>
				<li class="nav-item">
				  <a href="#" class="nav-link nav-icon">
					<svg width="1.5rem" height="1.5rem" fill="currentColor" aria-hidden="true" focusable="false" class="overflow-visible">
					  <use xlink:href="../Orange-Boosted-Bootstrap/site/static/docs/5.1/assets/img/boosted-sprite.svg#buy" />
					</svg>
					<span class="visually-hidden">Basket</span>
					<span class="position-relative">
					  <span class="badge bg-info rounded-pill position-absolute top-0 text-white translate-middle">
						1
						<span class="visually-hidden">shopping basket items</span>
					  </span>
					</span>
				  </a>
				</li>
				<li class="nav-item">
				  <a href="#" class="nav-link nav-icon">
					<img src="../Orange-Boosted-Bootstrap/site/static/docs/5.1/assets/img/navbar-contact.png" width="25" height="25" role="img" alt="User" loading="lazy">
					<span class="visually-hidden">My account</span>
				  </a>
				</li>
			  </ul>
			</div>
		  </div>
		</nav>
	  </header>
	  <div class="container">
		<div class="form-check form-switch py-5">
		  <input class="form-check-input" type="checkbox" role="switch" id="stickySwitch">
		  <label class="form-check-label" for="stickySwitch">Toggle sticky/fixed header</label>
		</div>
	  </div>
	</main>

    <!-- Option 1: Boosted Bundle with Popper -->
	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
    <script src="../Orange-Boosted-Bootstrap/dist/js/boosted.min.js"></script>
  </body>
</html>
```